_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -1553,26 +1575,6 @@ inputs = {
   # Cross-account IAM role permissions
   ##################################
 
-  # By granting access to the root ARN of the Security account in each of the roles below,
-  # we allow administrators to further delegate access to other IAM entities
-
-  # A role that allows administrator access to the account.
-  allow_full_access_from_other_account_arns = [local.security_account_root_arn]
-
-  # A role for developers to use to access services in the account.
-  # Access to services is managed using the dev_permitted_services input.
-  allow_dev_access_from_other_account_arns = [local.security_account_root_arn]
```

Comment:
Why remove this examples?

Reply:
Well this is for the logs account, and we don't really need these dev roles in there (what do we expect it to be useful for?). Or at least that was my understanding. What do you think?